tools/cndump/commands.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+#ifndef CNDUMP_COMMANDS_H
+#define CNDUMP_COMMANDS_H
+
+#define CNDB_COMMAND_NAME "cndb"
```

Comment:
All these name and description constants should probably just live in their respective headers.